gips/inventory/dbinv/models.py

```diff
@@ -26,12 +27,26 @@ class Asset(models.Model):
     tile   = models.TextField(db_index=True)   # 'h12v04'
     date   = models.DateField(db_index=True)   # of observation, not production
     name   = models.TextField()                # file name including full path
-    status = models.ForeignKey(Status)         #
-
+    status = models.TextField(validators=[valid_status])
+    
     class Meta:
         # These four columns uniquely identify an asset file
         unique_together = ('driver', 'asset', 'tile', 'date')
 
+    # TODO: this is clunky and makes multiple round trips, and is repetitive
+    #       seems to beg to be a trigger
```

Comment:
agree that this doesn't look ideal but it looks like it works and it's a prototype so :+1: to this method